docs/vpcanalyzer_lint.md

```diff
@@ -0,0 +1,93 @@
+## vpcanalyzer lint
+
+provides a detailed report of potential issues in the given VPC configuration
+
+### Synopsis
+
+The following linters are supported:
+
+| Name                            | Description                                                                              |
+|---------------------------------|------------------------------------------------------------------------------------------|
+| **nacl-split-subnet**           | Network ACLs implying different connectivity for endpoints inside a subnet               | 
+| **sg-split-subnet**             | SGs implying different connectivity for endpoints inside a subnet                        |  
+| **subnet-cidr-overlap**         | Overlapping subnet address spaces                                                        |  
+| **nacl-unattached**             | Network ACL not applied to any resources                                                 |  
+| **sg-unattached**               | SG not applied to any resources                                                          |  
+| **sg-rule-cidr-out-of-range**   | rules of security groups that reference CIDRs not in the relevant VPC address range      |
+| **nacl-rule-cidr-out-of-range** | rules of network Network ACLs that reference CIDRs not in the relevant VPC address range |
+| **tcp-response-blocked**        | Blocked TCP response                                                                     |
+| **nacl-rule-shadowed**          | rules of network Network ACLs that are shadowed by higher priority rules                 |
+| **sg-rule-implied**             | security group rules that are implied by other rules                                     |
```

Comment:
```suggestion
| **sg-rule-implied**             | Security-group rules implied by other rules                                     |
```

Reply:
Same with here. `Security-group` or `Security Group`? 
Again, lets be consistent in all appearances (not just in the readme)

Reply:
`Security Group`

Reply:
Neither IBM nor AWS use a hyphen in "Security Group" or "Network ACL", hence my preference. They could have, but chose not to...?

Reply:
Done (with "Security group")